blockchain/src/blockchain/mod.rs

```diff
@@ -1004,12 +1002,20 @@ impl AbstractBlockchain for Blockchain {
             .get_chunk(prefix, size, txn_option)
     }
 
+    fn get_batch_transactions(
+        &self,
+        batch: u32,
+        txn_option: Option<&Transaction>,
+    ) -> Option<Vec<BlockchainTransaction>> {
+        self.get_epoch_transactions(batch, txn_option)
+    }
+
```

Comment:
What does this actually do? It seems like it returns the transactions for epoch x when you ask for the transactions for batch x.

Reply:
This is part of the 1.0 chain object. Therefore there is no concept of epochs or batches and the functions threat the input as the actual block height, effectively making both functions exactly the same.

It is a remnant of the single repo for both versions of the protocol.

Reply:
Ok. Seems good to me then. We only need to solve the issue of failing when we provide any extra data to the macro blocks.